packages/website/ts/index.tsx

```diff
@@ -45,6 +45,21 @@ const LazySmartContractsDocumentation = createLazyComponent('Documentation', asy
 const LazyConnectDocumentation = createLazyComponent('Documentation', async () =>
     System.import<any>(/* webpackChunkName: "connectDocs" */ 'ts/containers/connect_documentation'),
 );
+const LazyWeb3WrapperDocumentation = createLazyComponent('Documentation', async () =>
+    System.import<any>(/* webpackChunkName: "connectDocs" */ 'ts/containers/web3_wrapper_documentation'),
```

Comment:
If we want all the docs to be in one chunk - let's name it in a more generic way

Reply:
Should we?

Reply:
`connectDocs` seems lik a chunk that has the docs for 0x-connect. But now it has most of them

Reply:
Ah I see. Left each as a separate chunk.